lldb/include/lldb/Core/Progress.h

```diff
@@ -149,6 +156,9 @@ class Progress {
 
   /// The "completed" value of the last reported event.
   std::optional<uint64_t> m_prev_completed;
+
+  /// The origin of this progress event.
+  ProgressOrigin m_origin;
```

Comment:
Pavel const'ified the things that cannot change, so we should mark this as `const` and move it up (before or after ProgressData) so these const members are grouped together.